docs/src/main/sphinx/functions/url.rst

```diff
@@ -40,7 +40,26 @@ such as ``:`` or ``?``.
 
 .. function:: url_extract_protocol(url) -> varchar
 
-    Returns the protocol from ``url``.
+    Returns the protocol from ``url``:
+    ::
+
+        SELECT url_extract_protocol('http://localhost:8080/req_path');
+        -- http
+
+        SELECT url_extract_protocol('https://127.0.0.1:8080/req_path');
+        -- https
+
+    If input ``url`` is not valid, then the behavior is undefined:
+    ::
+
+        SELECT url_extract_protocol('http://');
```

Comment:
I will ask if these are bugs or expected or so

Reply:
Sure, can you guide me where is implementation of the function? If it is a bug, I would try to fix this if possible.

Reply:
https://github.com/trinodb/trino/blob/master/core/trino-main/src/main/java/io/trino/operator/scalar/UrlFunctions.java#L57

here is the definition of the `url_extract_protocol` function.

If you follow the code you get to https://github.com/trinodb/trino/blob/master/core/trino-main/src/main/java/io/trino/operator/scalar/UrlFunctions.java#L191 which can be reproduced in `jshell`
to notice that the parsing will fail reason why `NULL` is being returned.

```
jshell> var uri = new java.net.URI("http://")
|  Exception java.net.URISyntaxException: Expected authority at index 7: http://
|        at URI$Parser.fail (URI.java:2913)
```

Reply:
@findinpath Thank you for your guidance

Reply:
I had a chat with @electrum and we might want to just remove this section for now. The behavior is kind of defined in the spec and we might want to change the implementation. So for the docs for now just remove the undefined sentence and examples. 

And it might be worth checking if ftp or so are supported..

Reply:
Thanks for explain, so i will remove the invalid examples and add a example about `ftp`